scripts/shell.sh

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env sh
+
+# Assumption #1: non-GNU, non-BSD linux distros are out-of-scope
+# Otherwise we at least need to get rid of 'local' keyword
```

Comment:
hm... we will want to support OSX eventually so might have to cater to that. Maybe up sh to a bash requirement?

Reply:
OSX is not a problem (yet, we'll see what apple comes up with next in their "security" frantic), nix was used by me and others extensively on MacOS, and I'm more then sure that the same setup will work on both. Granted not all packages might be available for OSX (surprise, surprise Linux open source community doesn't treat OSX releases and maintenance as a priority ;) ).

Reply:
I think you raise a valid point.
We are not clear in hardware/os we initially support, so I have added appropriate info in the readme. 
As to the MacOS compatibility, I would propose to refrain from adding changes which may be useful for the future - instead, let’s just do a separate PR which adds MacOS support.